### PR TITLE
(DOCSP-12903): Add a Realm Studio page to the top-level ToC

### PR DESCRIPTION
**Pull request by dacharyc**
### Jira

The linked Jira ticket is for providing a prominent place per SDK for Realm Studio, but we're giving it a prominent place at top-level ToC, so I think that satisfies this ticket. After we merge this, I'll add links to any SDK debugging pages to point to the top-level Realm Studio page.

- https://jira.mongodb.org/browse/DOCSP-12903

### Staged Changes (Requires MongoDB Corp SSO)

- [Realm Studio](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-12903/studio/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/studio.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.. _realm-studio:
+
+============
+Realm Studio
+============
+
+Get Started with Realm Studio
+-----------------------------
+
+.. include:: /includes/realm-studio.rst
```
Comment (nlarew) [inline source/studio.txt:10]: nit: do we use this include anywhere else anymore? If not, should we just inline it?

File: source/includes/realm-studio.rst [hunk 2]
```diff
@@ -6,8 +6,8 @@ Linux.
...
 
-See :github:`Releases on GitHub <realm/realm-studio/releases/tag/v10.0.0-beta.2>`:
+See :github:`Releases on GitHub <realm/realm-studio/releases/latest>`:
 
-- :github:`Download for Linux <realm/realm-studio/releases/download/v10.0.0-beta.2/realm-studio-10.0.0-beta.2.tar.gz>`
-- :github:`Download for Mac <realm/realm-studio/releases/download/v10.0.0-beta.2/MongoDB.Realm.Studio-10.0.0-beta.2-mac.zip>`
-- :github:`Download for Windows <realm/realm-studio/releases/download/v10.0.0-beta.2/MongoDB.Realm.Studio-10.0.0-beta.2-win.zip>`
+- `Download for Linux <https://studio-releases.realm.io/latest/download/linux-appimage>`__
+- `Download for Mac <https://studio-releases.realm.io/latest/download/mac-dmg>`__
+- `Download for Windows <https://studio-releases.realm.io/latest/download/win-setup>`__
```
Comment (nlarew) [inline source/includes/realm-studio.rst:13]:
```suggestion
- `Download for Windows <https://studio-releases.realm.io/latest/download/win-setup>`_
```
nit: let's prefer regular links (one underscore) over anonymous links (two underscores) - pretty sure anon links can lead to weird bugs on occasion

**Review by nlarew — APPROVED** [review]
LGTM w/ a couple nitty suggestions

File: source/includes/realm-studio.rst (hunk 2 above)
Comment (dacharyc) [reply source/includes/realm-studio.rst:13]: Thanks, @nlarew ! This style is the only one I could find in the RST Markup presentation I got during onboarding; I didn't know the single underscore was an option, or that there's a reason to prefer it. Updated!

File: source/studio.txt (hunk 1 above)
Comment (dacharyc) [reply source/studio.txt]: That's fair. Updated!

**Pull request merged**